service-front/config/autoload/global.php

```diff
@@ -123,14 +123,7 @@
     'email' => [
         // should reference a key within this array which provides
         // implementation-specific configuration
-        'transport' => getenv('OPG_LPA_FRONT_EMAIL_TRANSPORT') ?: 'sendgrid',
-
-        'sendgrid' => [
-            'key'     => getenv('OPG_LPA_FRONT_EMAIL_SENDGRID_API_KEY') ?: null,
-            'webhook' => [
-                'token' => getenv('OPG_LPA_FRONT_EMAIL_SENDGRID_WEBHOOK_TOKEN') ?: null,
-            ],
-        ], //sendgrid
+        'transport' => getenv('OPG_LPA_FRONT_EMAIL_TRANSPORT') ?: null,
```

Comment:
Can also remove this & all reference to `OPG_LPA_FRONT_EMAIL_TRANSPORT` now we've removed that logic

Reply:
Done